### PR TITLE
Employee dashboard

### PR DESCRIPTION
**Pull request by link5401**
What changed: 
Added task-submit form
Added dashboard for employee
Some jquery stuff for the dashboard (mostly front-end)
Some preps for back-end query

### CODE REVIEW

File: pages/components/employee-dashboard.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    }
+});
+
+$("#close-btn").click(function(event) {
+    $(".submit-card").hide();
+});
+
+//Handle back-end for submit btn with chosen target
+//Check target index with target.attr('id') (open-btn-index)
+//From index => get ID from index with taskIDs[index] above
```
Comment (link5401) [inline pages/components/employee-dashboard.js:44]: ID preps

File: pages/components/employee-dashboard.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+// Creating children: Query from database, create children element
+// and append too appropriate column
+
+
+
+////////////////////////////////////
+// Selecting children
```
Comment (link5401) [inline pages/components/employee-dashboard.js:7]: Expecting AJAX to query data from db. This section needs to:
+Query raw data
+Create jquery variable for #task-id, #task-desc, #deadline, #assigners
+Append data

File: pages/components/employee-dashboard.php [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    </div>
+    <div class="field-name col px-0" id="assigners"> Assigned by
+        <div class="element"> A</div>
+        <div class="element"> B</div>
+        <div class="element"> B</div>
+
+    </div>
+    <div class="field-name col px-0 action ">Action
+
+    </div>
```
Comment (link5401) [inline pages/components/employee-dashboard.php:30]: Div for auto-adding buttons

File: pages/employee.php [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    switch ($func) {
+        case 'employee-dashboard':
+            include_once 'components/employee-dashboard.php';
+            break;
+        case 'logout':
+            logout();
+            break;
+
+    }
+}
```
Comment (link5401) [inline pages/employee.php:13]: PHP for employee page

**Review by lwx270901 — APPROVED** [review]

**Pull request merged**